frontend/test/metabase/scenarios/question/reproductions/17910-revision-history-update.cy.spec.js

```diff
@@ -0,0 +1,34 @@
+import { restore, openOrdersTable, modal } from "__support__/e2e/cypress";
+
+describe("issue 17910", () => {
```

Comment:
We're experimenting with how we organize and run repro tests, could you please align these tests with the rest of the question repros, please?

1. Please make it export a function with test definition like in [other question repro files](https://github.com/metabase/metabase/tree/977ea2abcb61319125b82fa16d93c18610194134/frontend/test/metabase/scenarios/question/reproductions)
2. Replace `cy.spec.js` with just `.js`
3. Import and call the function from [this file](https://github.com/metabase/metabase/blob/977ea2abcb61319125b82fa16d93c18610194134/frontend/test/metabase/scenarios/question/question-reproductions.cy.spec.js)